cmd/manager/main.go

```diff
@@ -161,9 +164,33 @@ func main() {
 			},
 		}
 	}
+
+	// http/2 should be disabled due to its vulnerabilities. More specifically,
+	// disabling http/2 will prevent from being vulnerable to the HTTP/2 Stream
```

Comment:
```suggestion
	// disabling http/2 will prevent it from being vulnerable to the HTTP/2 Stream
```
Though more generally I think we could just replace the whole comment with: `http/2 should be disabled due to its vulnerabilities.` If they want to know more they can search the web.